### PR TITLE
Fix kubelet config comments

### PR DESCRIPTION
**Pull request by shuheiktgw**
#### What type of PR is this?
/sig node
/kind cleanup

#### What this PR does / why we need it:
While working on https://github.com/kubernetes/kubernetes/pull/105116 I noticed two minor problems in the comments
1. The URLs point to kube_features.go is wrong (probably old?)
2. The default value for `AllowedUnsafeSysctls` is wrong. It should be `nil` not `[]`

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

File: pkg/kubelet/apis/config/types.go [hunk 1]
```diff
@@ -326,7 +326,7 @@ type KubeletConfiguration struct {
 	IPTablesDropBit int32
 	// featureGates is a map of feature names to bools that enable or disable alpha/experimental
 	// features. This field modifies piecemeal the built-in default values from
-	// "k8s.io/kubernetes/pkg/features/kube_features.go".
+	// "https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go".
```
Comment (ehashman) [inline pkg/kubelet/apis/config/types.go:329]: This corrected link is not right. This isn't a link, it's the name to the fully qualified golang package.

File: staging/src/k8s.io/kubelet/config/v1beta1/types.go [hunk 2]
```diff
@@ -807,7 +807,7 @@ type KubeletConfiguration struct {
 	IPTablesDropBit *int32 `json:"iptablesDropBit,omitempty"`
 	// featureGates is a map of feature names to bools that enable or disable experimental
 	// features. This field modifies piecemeal the built-in default values from
-	// "k8s.io/kubernetes/pkg/features/kube_features.go".
+	// "https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go".
```
Comment (ehashman) [inline staging/src/k8s.io/kubelet/config/v1beta1/types.go:810]: Same as above.

File: staging/src/k8s.io/kubelet/config/v1beta1/types.go [hunk 3]
```diff
@@ -937,7 +937,7 @@ type KubeletConfiguration struct {
 	// A comma separated whitelist of unsafe sysctls or sysctl patterns (ending in `*`).
 	// Unsafe sysctl groups are `kernel.shm*`, `kernel.msg*`, `kernel.sem`, `fs.mqueue.*`,
 	// and `net.*`. For example: "`kernel.msg*,net.ipv4.route.min_pmtu`"
-	// Default: []
+	// Default: nil
```
Comment (ehashman) [inline staging/src/k8s.io/kubelet/config/v1beta1/types.go:940]: This also is incorrect. Both the internal and serializable versions of `AllowedUnsafeSysctls` are `[]string`, not a pointer type. `nil` is not a possible value.

**Review by ehashman — COMMENTED** [review]
/label api-review
/close

Neither of these changes are fixes; I've explained why in my review comments. Hence, I will close this PR.

Thanks for your efforts to contribute to Kubernetes! I encourage you to check out confirmed issues tagged `triage/accepted` or `help-wanted`.

**Comment by shuheiktgw** [comment]
@ehashman Thank you for your review! Now I understand it 👍

**Pull request closed without merging**